app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,47 @@
+class AttendancesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_attendance, only: [:show]
+  before_action :authorize_attendance, only: [:index, :new, :create]
+
+  def index
+    @attendances = policy_scope(Attendance.all).ordered
+    @attendance_today = current_user.attendances.find_by(date: Date.today) unless current_user.manager?
```

Comment:
```suggestion
    @attendance_today = current_user.attendances.find_by(date: Date.current) unless current_user.manager?
```
[Date.today vs Date.current](https://stackoverflow.com/questions/6635363/what-is-the-difference-between-date-current-and-date-today)